exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/CoreSchemaProxy.java

```diff
@@ -42,6 +50,13 @@
 
   private final NativeHandle nativeHandle;
   private final View dbView;
+  private static final Serializer<Block> blockSerializer = BlockSerializer.INSTANCE;
```

Comment:
it's a constant. Why not it the upper case?

Reply:
Yep, it falls under our definition: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names